Makefile

```diff
@@ -2,8 +2,11 @@ GOLINT := golangci-lint
 
 .PHONY: build test
 
+VERSION=$(shell git describe --tags --dirty --always)
+
 build:
-	go build -o conduit-connector-firebolt cmd/firebolt/main.go
+	go build -ldflags "-X 'github.com/conduitio-labs/conduit-connector-firebolt.version=${VERSION}'" -o conduit-connector-firebolt cmd/connector/main.go
+	cp conduit-connector-firebolt ${GOPATH}/src/github.com/conduitio/conduit/connectors/firebolt
```

Comment:
Why do we need this?

Reply:
To copy binary file  to  conduit connectors directory, it is easier then make it by hand

Reply:
But we can remove it, I think it is not necessary

Reply:
I think it's generally a good idea, but not everyone puts their repositories in the same location.

Reply:
I agree with you,  I removed it